README.md

```diff
@@ -337,6 +338,7 @@ ENV PHP_FPM_PM_MIN_SPARE_SERVERS="20"
 ENV PHP_FPM_PM_MAX_SPARE_SERVERS="40" 
 ENV PHP_FPM_PM_PROCESS_IDLE_TIMEOUT="35"
 ENV PHP_FPM_PM_MAX_REQUESTS="500"
+PHP_FPM_ACCESS_FORMAT={"C":"%C","d":"%d","f":"%f","l":"%l","m":"%m","M":"%M","n":"%n","P":"%P","p":"%p","q":"%q","Q":"%Q","r":"%r","R":"%R","s":"%s","T":"%T","t":"%t","u":"%u"}
```

Comment:
Should it be prefixed with `ENV` like the ones above it?

Reply:
Yes it should, and is now done